### ISSUE TITLE
Put a screenshot on readme.md

### ISSUE THREAD

**Issue by bovine3dom** [tridactyl/tridactyl#540]
Projects without screenshots are a little pet hate of mine, so it's only natural that we forgot to put a screenshot on the readme...

**Comment by glacambre** [comment]
We could even have gifs showcasing each feature Tridactyl has. I believe this would be useful because Tridactyl's features aren't easy to discover right now.
But adding multiple pictures to the repo might not be a good idea, in my opinion the gifs would need to be uploaded elsewhere.

**Comment by cmcaine** [comment]
The gif is horrendously ugly. Do you still have the jpegs that made it? I'd like to try and do a better job.

**Comment by bovine3dom** [comment]
Yeah, I do. It's just in ~/Videos on my laptop. Remind me and I'll give them to you.

**Comment by cmcaine** [comment]
Ta. I don't know if I'll be able to do any better, but I'd like to try.

**Comment by glacambre** [comment]
Re-shooting the video and saving to PNG instead of JPG might help as the video contains multiple large spaces with a single color. It'd also let you run something like `pngcrush -brute` on each frame which would certainly reduce file size. Not sure the result would be worth the effort though.

**Comment by bovine3dom** [comment]
No need to reshoot - I have the MP4 and it's very crisp.

**Comment by cmcaine** [comment]
note to self the ffmpeg commands here might be useful https://gist.github.com/Ruin0x11/8fae0a9341b41015935f76f913b28d2a

**Comment by bovine3dom** [comment]
@cmcaine you should remind me this evening : )

**Comment by glacambre** [comment]
I recently tried to turn a screen recording into a gif and doing it "naively" with a call to ffmpeg resulted in a 13MB file. Fiddling with imagemagick resulted in a 50KB file which looked much better. I recorded the video in .apng format and then ran the following commands:
```sh
ffmpeg -i video.apng -r 10/1 frame_%03d.png
convert -dither None -layers Optimize frame_*.png video.gif
```
I'm convinced `-dither None` would help with quality here. Not sure about file size though.
